Automattic/wp-calypso
# Themes: Remove Unused Current Theme Button

#### Changes proposed in this Pull Request

* Removes unused Current Theme Button file

#### Testing instructions



* Apply this PR
* Navigate to Appearance > Themes
* Verify that the current theme card at the top of the page and its associated actions still behave as expected



https://github.com/Automattic/wp-calypso/issues/60530
